spec/pubid_bsi/identifier_spec.rb

```diff
@@ -100,5 +100,35 @@ module Pubid::Bsi
 
       it_behaves_like "converts pubid to pubid"
     end
+
+    context "adopted documents" do
+      context "PD IEC/TR 80002-3:2014" do
+        let(:original) { "PD IEC/TR 80002-3:2014" }
+        let(:pubid) { "PD IEC TR 80002-3:2014" }
+
+        it_behaves_like "converts pubid to pubid"
+
+        it { expect(subject.adopted).to be_a(Pubid::Iec::Base) }
+        it { expect(subject.adopted.to_s).to eq("IEC TR 80002-3:2014") }
+      end
+
+      context "BS ISO/IEC 30134-1:2016" do
+        let(:pubid) { "BS ISO/IEC 30134-1:2016" }
+
+        it_behaves_like "converts pubid to pubid"
+
+        it { expect(subject.adopted).to be_a(Pubid::Iec::Base) }
+        it { expect(subject.adopted.to_s).to eq("ISO/IEC 30134-1:2016") }
+      end
+
+      context "BS ISO/PRF PAS 5643" do
+        let(:original) { "BS ISO/PRF PAS 5643" }
+        let(:pubid) { "BS ISO/PAS 5643" }
```

Comment:
This is incorrect, because the "ISO/PRF PAS" is a typed-staged identifier. The correct pubid is "BS ISO/PRF PAS 5643"

Reply:
> This is incorrect, because the "ISO/PRF PAS" is a typed-staged identifier. The correct pubid is "BS ISO/PRF PAS 5643"

By default, the "PRF" stage hidden by default because of https://github.com/metanorma/pubid-iso/issues/106
With additional rendering options (`iso_pubid.to_s(with_prf: true)`) we could get identifier with "PRF", I can update tests to do this, but I believe we will never see BSI identifiers with adopted ISO identifiers with "PRF" stage. I used this identifier just to make sure the "ISO" parser works well.

Reply:
It would be good to update tests to do this. 

These are some examples from resellers of BSI content that give the BS document an ISO stage:
* "BS ISO/DIS 22000:2017" https://standard.no/no/Nettbutikk/produktkatalogen/Produktpresentasjon/?ProductID=917508
* "BS ISO/DIS 9004.1:2017" https://www.techstreet.com/standards/bs-iso-dis-9004-1-2017?product_id=1997900
* "BS ISO/FDIS 22301:2012"  https://www.techstreet.com/standards/bs-iso-fdis-22301-2012?product_id=1826533